### PR TITLE
16 tariff edit

### PR DESCRIPTION
**Pull request by romigos**

### CODE REVIEW

File: app/views/tariffs/_form.html.erb [hunk 1]
```diff
@@ -1,7 +1,11 @@
 <%= form_for :tariff, url: tariffs_path do |f| %>
   <p>
-    <%= f.label :name, 'Назва тарифу' %>
-    <%= f.text_field(:name, { class: 'form-control' }) %>
+    <%= f.label :format_photo_id, 'Група' %>
+   <%= f.collection_select :tariff_group_id, TariffGroup.all, :id, :name,  {label: "Група тарифів", prompt: "Виберіть групу тарифів"}  %>
+  </p>
+  <p>
+    <%= f.label :tariff_format_id, 'Формат' %>
+    <%= f.collection_select :format_photo_id, FormatPhoto.all, :id, :name,  {label: "Формат фото", prompt: "Виберіть формат фото"}  %>
```
Comment (saviokmua) [inline app/views/tariffs/_form.html.erb:8]: @romigos please move `FormatPhoto.all` to value of controller(like `format_photos = FormatPhoto.all`) and use it
also please do for TariffGroup.all

File: app/views/tariffs/_form.html.erb [hunk 2]
```diff
@@ -1,7 +1,11 @@
 <%= form_for :tariff, url: tariffs_path do |f| %>
   <p>
-    <%= f.label :name, 'Назва тарифу' %>
-    <%= f.text_field(:name, { class: 'form-control' }) %>
+    <%= f.label :format_photo_id, 'Група' %>
```
Comment (saviokmua) [inline app/views/tariffs/_form.html.erb]: f.label :tariff_group_id

File: app/views/tariffs/_form.html.erb [hunk 3]
```diff
@@ -1,7 +1,11 @@
...
+   <%= f.collection_select :tariff_group_id, TariffGroup.all, :id, :name,  {label: "Група тарифів", prompt: "Виберіть групу тарифів"}  %>
+  </p>
+  <p>
+    <%= f.label :tariff_format_id, 'Формат' %>
```
Comment (saviokmua) [inline app/views/tariffs/_form.html.erb]: f.label :format_photo_id ...

File: app/views/tariffs/index.html.erb [hunk 4]
```diff
@@ -2,13 +2,16 @@
...
+    <td width="30%">Група</td>
+    <td width="30%">Формат</td>
+    <td width="30%">Ціна</td>
+    <td colspan="1" width="150"><%= link_to 'Додати', new_tariff_path, class: 'btn btn-warning' %></td>
+    <td colspan="1" width="150"><%= link_to 'Повернутись', root_path, class: 'btn btn-warning' %></td>
   </tr>
   <% @tariff.each do |tariff| %>
     <tr>
-      <td><%= tariff.name %></td>
+      <td><%= tariff.format_photo_id %></td>
```
Comment (saviokmua) [inline app/views/tariffs/index.html.erb]: @romigos is it worked correctly?

**Review by saviokmua — CHANGES_REQUESTED** [review]
please test all

File: app/views/tariffs/_form.html.erb (hunk 2 above)
Comment (romigos) [reply app/views/tariffs/_form.html.erb]: fixed

File: app/views/tariffs/_form.html.erb (hunk 3 above)
Comment (romigos) [reply app/views/tariffs/_form.html.erb]: fixed

File: app/views/tariffs/index.html.erb (hunk 4 above)
Comment (romigos) [reply app/views/tariffs/index.html.erb]: fixed

File: app/views/tariffs/_form.html.erb (hunk 1 above)
Comment (romigos) [reply app/views/tariffs/_form.html.erb:8]: i do not understand all is working

**Comment by romigos** [comment]
all is tested

**Pull request merged**